ee/fleet-agent-downloader/api/hooks/sso/index.js

```
@@ -47,54 +47,54 @@ module.exports = function (sails){
           routes: {
             login: {
               path: '/login',
             },
             loginCallback: {
               path: '/authorization-code/callback',
-              afterCallback: '/entrance/signup-okta-user-or-redirect',
+              afterCallback: '/entrance/signup-sso-user-or-redirect',
             },
             logout: {
-              path: '/okta/logout'
+              path: '/sso/logout'
```

ESLint

[error] Replace `'/sso/logout'` with `"/sso/logout",`

(prettier/prettier)